pkg/api/api.go

```diff
@@ -165,5 +185,8 @@ func ConfigureAPI(treeID uint) {
 }
 
 func StopAPI() {
+	if api.newEntryPublisher != nil {
+		api.newEntryPublisher.Close()
```

Comment:
Does this wait until all transactions are complete?

Reply:
It would be up to each publisher implementation. The current GCP implementation does not wait.

Reply:
Can we add waiting? Minimizes the risk of currently processing messages from not being sent.

Reply:
Done